sorc/build_ufs.sh

```diff
@@ -32,6 +32,8 @@ source "./tests/module-setup.sh"
 MAKE_OPT="-DAPP=${APP} -D32BIT=ON -DCCPP_SUITES=${CCPP_SUITES}"
 [[ ${PDLIB:-"OFF"} = "ON" ]] && MAKE_OPT+=" -DPDLIB=ON"
 [[ ${BUILD_TYPE:-"Release"} = "Debug" ]] && MAKE_OPT+=" -DDEBUG=ON"
+#MAKE_OPT+=" -DFASTER=ON"  #comment out this line for the global_nest runs
+
```

Comment:
-f command line option can be easily passed into build_ufs.sh but it is hard to be passed into build_all.sh
We will consider a full workflow instead of only the forecast part.
Users need to modify build_ufs.sh anyway to include the global_nest_v1 physics in the compiling time.

Reply:
In that case, it is best to use -DFASTER=ON as the default. It was designed to be both faster *and safer* than the defaults.

Reply:
I trust you. But I am not sure whether EMC will accept a default -DFASTER=ON.  But we can try.